doc/configuration.rst

```diff
@@ -411,6 +411,35 @@ hive resources and 1 mysql resource:
 Note that it was not necessary to specify the 1 for mysql here, but it
 is good practice to do so when you have a fixed set of resources.
 
+.. _retcode-config:
+
+[retcode]
+----------
+
+Configure return codes for the luigi binary. In the case of multiple return
```

Comment:
Yeah, I do understand both of the black bullet points. My question was different - I'm questioning why we seem to encourage the concrete values of "different return codes" to vary from one environment to another. Better standardize on one meaning of each return code across all Luigi community?